exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,5 @@
 const name = " Daniel  ";
-
+message = name.trim(" ");
```

Comment:
almost - variables in JS need *declaring* by prefixing the variable name with `let` or `const`. This effectively tells JS "Hey, every time you see `variableName` from this point onwards, it's actually talking about this variable"